go.mod

```diff
@@ -26,14 +26,16 @@ require (
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/shurcooL/githubv4 v0.0.0-20220520033151-0b4e3294ff00
 	github.com/xanzy/go-gitlab v0.73.1
-	golang.org/x/text v0.7.0
+	golang.org/x/text v0.14.0
 	gopkg.in/ldap.v2 v2.5.1
 	k8s.io/api v0.24.2
 	k8s.io/apimachinery v0.24.2
 	k8s.io/client-go v0.24.2
 	sigs.k8s.io/controller-runtime v0.12.2
 )
 
+require github.com/golang-jwt/jwt/v5 v5.0.0 // indirect
```

Comment:
Can we add this into the existing `require` block

Reply:
updated go.mod